### PR TITLE
custom armor

### PR DESCRIPTION
**Pull request by OverYandere**
The new MA ODST armor has been implemented. Includes: CH252D Helmet in 5 camo variants with 3 visor colors, M52 Vest in 5 camo patterns with toggle-able medic sigil for standard shoulders as well as CQB and Marksman option for all 5 camos, shoulder-less marine uniform to go with the new armor included in 6 camo patterns.

Custom Armor framework coming in the next batch.

### CODE REVIEW

**Comment by Lupus590** [comment]
It looks great however there is some issues with shoulders, check your file paths for textures. Also, I still need to look at the code.
image
image

**Comment by Lupus590** [comment]
Misriah armoury updated and I think it affects this pull request.

**Comment by Lupus590** [comment]
Also, can the new ODST stuff go in a new folder `odst2`?

**Comment by Lupus590** [comment]
Also, can we split the custom armour and the new ODST stuff into separate pull request?

**Comment by Lupus590** [comment]
blocked by #230

**Review by Lupus590 — CHANGES_REQUESTED** [review]

File: src/Addons/34thPRC_ArmourCustom/config.cpp [hunk 1]
```diff
@@ -6,6 +6,7 @@ class CfgPatches
 		units[]={};
 		requiredAddons[]=
 		{
+			//"MA_Armor", //MA_Armor
```
[the remark this answers is not in the document]
Comment (OverYandere) [reply src/Addons/34thPRC_ArmourCustom/config.cpp:9]: I forgot to take it out during attempts to get depolarized working on ArmourCustom. There was an issue with ItemInfo not being defined despite inheriting from ArmourStandard where it works so I tried to go 1 up and inherit from MA_Armor which also didn't work. I ended up having to do class ItemInfo; in the main cfg and that worked.

[the remark this answers is not in the document]
Comment (Lupus590) [reply src/Addons/34thPRC_ArmourCustom/config.cpp:9]:
```suggestion

```

[the remark this answers is not in the document]
Comment (OverYandere) [reply src/Addons/34thPRC_ArmourCustom/config.cpp:9]: Deleted in coming patch

File: src/Addons/34thPRC_ArmourCustom/data/odst2/config_weapons.hpp [hunk 2]
```diff
@@ -0,0 +1,2 @@
+//Include each player's config
```
[the remark this answers is not in the document]
Comment (OverYandere) [reply src/Addons/34thPRC_ArmourCustom/data/odst2/config_weapons.hpp:1]: Done in next commit

File: src/Addons/34thPRC_ArmourCustom/data/marines2/config_weapons.hpp [hunk 3]
```diff
@@ -0,0 +1,2 @@
+//Put player includes here
```
[the remark this answers is not in the document]
Comment (OverYandere) [reply src/Addons/34thPRC_ArmourCustom/data/marines2/config_weapons.hpp:1]: Done in next commit

File: src/Addons/34thPRC_AceArsenalExtended/data/aax_models/weapons/armour_standard/marines2/config.hpp [hunk 4]
```diff
@@ -1,169 +1,169 @@
...
+		class true
+		{
+			label = "Medic";
+		};
+	};
+	class eyewear_color
+
+	{
+
+		alwaysSelectable = 1;
```
[the remark this answers is not in the document]
Comment (OverYandere) [reply src/Addons/34thPRC_AceArsenalExtended/data/aax_models/weapons/armour_standard/marines2/config.hpp:145]: Done in next commit

File: src/Addons/34thPRC_AceArsenalExtended/data/aax_models/weapons/armour_standard/marines2/config.hpp [hunk 5]
```diff
@@ -1,169 +1,169 @@
...
+};
+
+// MA CH252 Helmets
+class 34thPRC_ArmourStandard_Marines2_CH252_Urban
+{
+	label = "[34thPRC] CH252 Helmets";
+	author = "34th PRC Modding Team, J. Koko";
+	options[] = {"camo", "eyewear", "medic", "eyewear_color"};
+
+	class camo
```
[the remark this answers is not in the document]
Comment (OverYandere) [reply src/Addons/34thPRC_AceArsenalExtended/data/aax_models/weapons/armour_standard/marines2/config.hpp:83]: Done in next commit

File: src/changelog.md [hunk 6]
```diff
@@ -16,11 +16,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## 0.17.0
 ### Added
-- 5 color variants for the marine helmet eyewear
```
[the remark this answers is not in the document]
Comment (OverYandere) [reply src/changelog.md:20]: Should be reverted now

**Review by Lupus590 — APPROVED** [review]

File: .gitattributes [hunk 7]
```diff
@@ -8,3 +8,4 @@
 *.hpp text eol=crlf
 *.cpp text eol=crlf
 *.rvmat text eol=crlf
+* -text
```
Comment (Lupus590) [inline .gitattributes:11]: We might be able to remove this

Comment (Lupus590) [reply .gitattributes:11]:
```suggestion

```

**Review by Lupus590 — APPROVED** [review]

**Review by Lupus590 — APPROVED** [review]

**Pull request merged**